parsers/ES.py

```diff
@@ -216,13 +225,6 @@ def fetch_production(
                     response_data["production"]["hydro"] = 0.0
                     response_data["storage"]["hydro"] = -response.hydraulic
 
-                response_data = validate(
-                    response_data,
-                    logger,
-                    floor=ZONE_FLOORS[zone_key],
-                    expected_range=expected_range,
-                )
-
```

Comment:
This was not really used in this parser so to avoid adding back the entire mapping table (that was mostly 0) I removed it.